### PR TITLE
More fixes and updates

### PR DESCRIPTION
**Pull request by redhatrises**
- More pep8 fixes
- Add env variables handling
- Add some defaults for cli args
- Add support for older docker py API

### CODE REVIEW

File: cs_scanimage.py [hunk 1]
```diff
@@ -207,26 +209,57 @@ def __call__(self, r):
...
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, values)
+# End code authored by Russell Heilling
+
+
 def parse_args():
-    parser = argparse.ArgumentParser(description='Crowdstrike scan your docker image.')
+    parser = argparse.ArgumentParser(
+        description='Crowdstrike - scan your Docker image.')
```
Comment (shawndwells) [inline cs_scanimage.py:231]:
```suggestion
        description='Crowdstrike - scan your container image.')
```

**Review by shawndwells — CHANGES_REQUESTED** [review]

**Comment by redhatrises** [comment]
@isimluk thoughts?

**Comment by isimluk** [comment]
Would you be willing to use FALCON_CLIENT_ID, FALCON_CLIENT_SECRET, and FALCON_CLOUD as environment variables?

I am trying to get us towards some consistency, so I always use these 3 names.

**Review by isimluk — APPROVED** [review]
👍 LGTM, like the direction!

**Comment by redhatrises** [comment]
> Would you be willing to use FALCON_CLIENT_ID, FALCON_CLIENT_SECRET, and FALCON_CLOUD as environment variables?
> 
> I am trying to get us towards some consistency, so I always use these 3 names.

Works for me... I just hate typing those in... that's all. :D

**Comment by redhatrises** [comment]
> > Would you be willing to use FALCON_CLIENT_ID, FALCON_CLIENT_SECRET, and FALCON_CLOUD as environment variables?
> > I am trying to get us towards some consistency, so I always use these 3 names.
> 
> Works for me... I just hate typing those in... that's all. :D

Updated the variables. Failing tests can be ignored until we can determine the best fixes for them.

**Comment by isimluk** [comment]
LGTM. 👍 💯

**Comment by redhatrises** [comment]
Thanks @isimluk !

**Pull request merged**